### PR TITLE
Added TypeScript implementation

### PR DESCRIPTION
**Pull request by Kwehdev**
TypeScript implementation of Fibonacci sequence.

### CODE REVIEW

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 7
           

Complexity increasing per file
==============================
- typescript_fibonacci.ts  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784&bid=20460283)

File: typescript_fibonacci.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const getFibonacciNumbers = (seriesLength: number): number[] => {
+  //Will always start with 0, 1, so we can safely initialize.
+  const seriesArr: number[] = [0, 1]
+
+  //We added the two initial numbers, so remove 2 from series length to compensate.
+  for (let i = 1; i <= seriesLength - 2; i++) {
+    seriesArr.push(seriesArr[i - 1] + seriesArr[i])
+  }
+  return seriesArr
+}
```
Comment (fharookshaik) [inline typescript_fibonacci.ts:10]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784)

File: typescript_fibonacci.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+console.log(getFibonacciNumbers(50))
```
Comment (fharookshaik) [inline typescript_fibonacci.ts:12]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784)

Comment (fharookshaik) [inline typescript_fibonacci.ts:12]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784)

File: typescript_fibonacci.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const getFibonacciNumbers = (seriesLength: number): number[] => {
+  //Will always start with 0, 1, so we can safely initialize.
+  const seriesArr: number[] = [0, 1]
+
+  //We added the two initial numbers, so remove 2 from series length to compensate.
+  for (let i = 1; i <= seriesLength - 2; i++) {
+    seriesArr.push(seriesArr[i - 1] + seriesArr[i])
```
Comment (fharookshaik) [inline typescript_fibonacci.ts:7]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784)

File: typescript_fibonacci.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const getFibonacciNumbers = (seriesLength: number): number[] => {
+  //Will always start with 0, 1, so we can safely initialize.
+  const seriesArr: number[] = [0, 1]
```
Comment (fharookshaik) [inline typescript_fibonacci.ts:3]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784)

File: typescript_fibonacci.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+  //We added the two initial numbers, so remove 2 from series length to compensate.
+  for (let i = 1; i <= seriesLength - 2; i++) {
+    seriesArr.push(seriesArr[i - 1] + seriesArr[i])
+  }
+  return seriesArr
```
Comment (fharookshaik) [inline typescript_fibonacci.ts:9]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784)

File: typescript_fibonacci.ts (hunk 3 above)
Comment (fharookshaik) [inline typescript_fibonacci.ts:7]: Codacy found an issue: [Generic Object Injection Sink](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784)

**Comment by Kwehdev** [comment]
Semicolons are not required in JS/TS, typically when working on a project a configuration is adopted and whether to use semicolons or not would be decided then, whether to use them or not is simply a styling choice. I can add them if you would prefer, but it's a non-issue.

Generic Object Injection does not apply here, because 'i' is not accessible to the user.

The console.log() is to demonstrate usage when running.

File: typescript_fibonacci.ts (hunk 3 above)
Comment (Kwehdev) [reply typescript_fibonacci.ts:7]: Generic Object Injection does not apply here, because 'i' is not accessible to the user.

**Comment by fharookshaik** [comment]
Nah! That's okay. The error messages are automatically generated by the bot. It'll recommend changes which improves the code readability.
If you can resolve the errors, that's fine. Otherwise make a comment down here. Your PR will be merged. 
Please add your name and the programming language inside the specific .md files.

Happy Coding!!
Happy Contributing!!

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- typescript_fibonacci.ts  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784&bid=20460283)

File: typescript_fibonacci.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
+const getFibonacciNumbers = (seriesLength: number): number[] => {
+  //Will always start with 0, 1, so we can safely initialize.
+  const seriesArr: number[] = [0, 1];
+
+  //We added the two initial numbers, so remove 2 from series length to compensate.
+  for (let i = 1; i <= seriesLength - 2; i++) {
+    seriesArr.push(seriesArr[i - 1] + seriesArr[i]);
```
Comment (fharookshaik) [inline typescript_fibonacci.ts:7]: Codacy found an issue: [Generic Object Injection Sink](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784)

**Comment by Kwehdev** [comment]
> Nah! That's okay. The error messages are automatically generated by the bot. It'll recommend changes which improves the code readability.
> If you can resolve the errors, that's fine. Otherwise make a comment down here. Your PR will be merged.
> Please add your name and the programming language inside the specific .md files.
> 
> Happy Coding!!
> Happy Contributing!!

Sure;
I've replaced the line endings with semicolons & Removed the console log statement. I think it'll still complain about Generic Object Injection Sink, but aside from that, 👍 .

I will also update the mentioned .md file.

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- typescript_fibonacci.ts  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784&bid=20460283)

File: typescript_fibonacci.ts (hunk 6 above)
Comment (fharookshaik) [inline typescript_fibonacci.ts:7]: Codacy found an issue: [Generic Object Injection Sink](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784)

**Comment by fharookshaik** [comment]
Please provide the output screenshot of your working code

**Comment by fharookshaik** [comment]
I request you to provide the screenshot of the code that you've written before others PR is merged.

**Comment by Kwehdev** [comment]
Capture

**Comment by fharookshaik** [comment]
Thank You for your Contribution!!

**Pull request merged**